src/components/LoginPage/LoginPageComponent.js

```diff
@@ -30,14 +30,10 @@ const H3 = styled.h3`
   color: ${COLORS.white};
 `;
 
-// 2% - indent for image positioning
-// 92% - image height without vertical indention
-// 4% - vertical indention
-// 0.75% - horizontal indention
-const GoogleLoginButton = styled.button`
+const buttonIn = css`
   color: ${COLORS.white};
   background-color: ${COLORS.blue};
-  margin: auto;
+  float: right;
```

Comment:
why changing float?
Won't the sign up button at the bottom the page be affected by this change? I.e. Move to the right from center?